src/user-icon/user-icon.scss

```diff
@@ -4,57 +4,47 @@
 @import '../style/index';
 @import '../text/mixins';
 
-@mixin iui-user-icons($size: $iui-icons-xl) {
+@mixin iui-user-icon {
   @include iui-reset;
-
   @include iui-font-family;
   display: inline-block;
   position: relative;
-  width: $size;
-  height: $size;
   margin: 0;
   user-select: none;
 
-  > .iui-user-icons-stroke {
+  > .iui-stroke {
```

Comment:
I think we could come up with a better name than "stroke" 🤔

Reply:
I'm used to the term stroke because it's [used in Photoshop](https://helpx.adobe.com/photoshop-elements/using/fills-strokes.html#stroke_outline_objects_on_a_layer).  How about `iui-border`?